Neos.ContentGraph.DoctrineDbalAdapter/src/DoctrineDbalContentGraphSchemaBuilder.php

```diff
@@ -142,4 +175,21 @@ private function createRestrictionRelationTable(Schema $schema): void
                 'affectednodeaggregateid'
             ]);
     }
+
+    private function createSchemaConfiguration(AbstractSchemaManager $schemaManager): SchemaConfig
+    {
+        // We should still grab the default connection schema to get eg. plattform identifier length limits
+        $schemaConfiguration = $schemaManager->createSchemaConfig();
+        $schemaConfig = new SchemaConfig();
+        $schemaConfig->setDefaultTableOptions([
+            'charset' => 'utf8mb4'
+        ]);
+
+        return $schemaConfig;
+    }
+
+    private function getDefaultTextCollation(): string
+    {
+        return 'utf8mb4_unicode_520_ci';
+    }
```

Comment:
Just wondering: Does this have to be an instance method? It could be static, but I would suggest to turn it into a private constant instead.
And don't worry for future changes: It is very local and only used in two places. If this happens to be dependent on the connection at some point, we can always change that lateron

Reply:
yeah exactly willdo